ml-agents/mlagents/trainers/stats.py

```diff
@@ -279,14 +299,25 @@ def write_stats(self, step: int) -> None:
 
     def get_stats_summaries(self, key: str) -> StatsSummary:
         """
-        Get the mean, std, and count of a particular statistic, since last write.
+        Get the mean, std, sum, count and aggregation method of a particular statistic, since last write.
+
         :param key: The type of statistic, e.g. Environment/Reward.
-        :returns: A StatsSummary NamedTuple containing (mean, std, count).
+        :returns: A StatsSummary NamedTuple containing (mean, std, sum, count, aggregation).
         """
         if len(StatsReporter.stats_dict[self.category][key]) > 0:
-            return StatsSummary(
-                mean=np.mean(StatsReporter.stats_dict[self.category][key]),
-                std=np.std(StatsReporter.stats_dict[self.category][key]),
-                num=len(StatsReporter.stats_dict[self.category][key]),
-            )
+            if (
```

Comment:
It's not your fault, but this method is getting messy and should be cleaned up. Something like
```python
stat_values = StatsReporter.stats_dict[self.category][key]
if len(stat_values) == 0:
    return StatsSummary.empty()

return StatsSummary(
    mean=np.mean(stat_values),
    sum=np.sum(stat_values),
    std=np.std(stat_values),
    num=len(stat_values),
    aggregation_method = StatsReporter.stats_aggregation[self.category][key],
)
```
should be a bit cleaner.